homeassistant/components/notify/telegram.py

```diff
@@ -109,7 +109,8 @@ def send_message(self, message="", **kwargs):
         # send message
         try:
             self.bot.sendMessage(chat_id=self._chat_id,
-                                 text=title + "  " + message)
```

Comment:
Telegram is not alone in having the title problem. The real solution would be to default the title in the notify component to `None` and have each platform default it to 'Home Assistant' if they support titles. Right now because we don't know if a title is default, each platform has to include it.

This would require an extra check for each platform because they now assume a title is always present.